RadarSDK/RadarLogBuffer.h

```diff
@@ -0,0 +1,31 @@
+//
+//  RadarLogBuffer.h
+//  RadarSDK
+//
+//  Copyright © 2021 Radar Labs, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "Radar.h"
+#import "RadarLog.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface RadarLogBuffer : NSObject
+
+- (instancetype)init;
+
```

Comment:
Add a comment that says the `sharedInstance` should be used instead, or move this declaration to the private interface.